include/functions.h

```diff
@@ -1430,8 +1430,10 @@ void Matrix_RotateZYX(s16 x, s16 y, s16 z, u8 mode);
 void Matrix_TranslateRotateZYX(Vec3f* translation, Vec3s* rotation);
 void Matrix_SetTranslateRotateYXZ(f32 translateX, f32 translateY, f32 translateZ, Vec3s* rot);
 Mtx* Matrix_MtxFToMtx(MtxF* src, Mtx* dest);
-Mtx* Matrix_ToMtx(Mtx* dest, char* file, s32 line);
-Mtx* Matrix_NewMtx(GraphicsContext* gfxCtx, char* file, s32 line);
+Mtx* Matrix_ToMtx(Mtx* dest);
+Mtx* Matrix_ToMtxDebug(Mtx* dest, char* file, s32 line);
+Mtx* Matrix_NewMtx(GraphicsContext* gfxCtx);
+Mtx* Matrix_NewMtxDebug(GraphicsContext* gfxCtx, char* file, s32 line);
```

Comment:
Confused on your comment in the PR description about matrix functions. You made two whole new functions and you arent sure if they exist? Feel like we should confirm something like this before putting it in

Reply:
The debug ROM and the retail ROM both have two functions here. The debug ROM functions take additional `file` and `line` arguments while the retail versions don't. So I'm not sure whether to name both versions the same thing, or add a `Debug` suffix to the debug versions like for the other macros

Reply:
Right now I kinda feel like they should be named the same cause of how they get swapped out. but more opinions needed

Reply:
I would either not introduce the non-debug variants for now (using `Matrix_ToMtxDebug(..., "", 0)` for now to define the non-debug macro), or wait for a version to be worked on in the repo that has these non-debug variants

Reply:
Those functions are swapped between debug and non debug they should probably be named the same.
Also I don't think it is a good idea to add new functions on a PR that was meant to create macros

Reply:
Okay, I've named the non-debug and debug functions the same